modules/ROOT/pages/configure-multiple-credential-providers.adoc

```diff
@@ -0,0 +1,80 @@
+= Configuring Multiple Credential Providers
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: credential providers, external client providers
+
+With Anypoint Platform, you can use multiple client providers, also known as identity providers (IdP) to manage client credentials to help you implement security policies and regulations. This ensures that client applications that are created to access APIs are stored in different systems based on the environment where API is located. Separate client providers helps with segregating client application management for:
+
+* Different business groups
+* Different environments, such as production, QA, and sandbox
+* Internal and external APIs
+
+For example, a “sandbox” client, whose credentials live in an external Okta IdP, can have contracts only with API instances in the organization “Sandbox A” environment of the business group A and “open” environment of business group B. Similarly, a “core-internal” client, whose credentials live in the private ADFS IdP of the organization, can have contracts only with API instances in the “prod” environment of the business group X. 
+
+
+== How This Works
+
+When you enable external IdPs, they act as authorization servers to issue access tokens to a client application for accessing an API. When you request access to a client application, the corresponding client ID and client secret credentials are created in the IdP specified in that API specification.
+
+If an API has only applied the Client ID Enforcement policy and not applied the OAuth policy the client application is still created in an external IdP.
+The API owner does not have access to the IdP configuration in API Manager. 
+
+When an OAuth policy is deployed to API gateway, the platform automatically injects IdP configuration, such as the token URL and token introspection url in the policy configuration. The IdP configuration is completely hidden from a user in API Manager.
+
+If no external IdPs are enabled, Access Manager acts as the default authorization server in that case. 
+//[PA:-Tomas not sure what the following statement is trying to say.]
+// Any client apps (clientID/clientSecret) that’re created for accessing an API are created and maintained within Anypoint.
+// OOTB OAuth policies with an external IdP can’t be applied 
+// ClientId enforcement policy can be applied.
+
+
+== The Use Case
```

Comment:
@priyaayyar @tbruno Is this the correct use case? It doesn't look like a use case of client management instead it looks a use case of single-sign-on